argo/application-field-validation/01-install.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- crd.yaml
+- application-field-validation.yaml
```

Comment:
Where is this file?

Reply:
I didn't get it. Isn't this the file that needs to be created?

Reply:
It already exists. Check the repo.

Reply:
https://github.com/kyverno/policies/blob/main/argo/application-field-validation/application-field-validation.yaml

Reply:
My bad, I thought this was a new set of the kuttl test.